samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.monitoring;
+
+// [START monitoring_create_alert_policy]
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.monitoring.v3.AlertPolicyServiceClient;
+import com.google.monitoring.v3.Aggregation;
+import com.google.monitoring.v3.AlertPolicy;
+import com.google.monitoring.v3.ComparisonType;
+import com.google.monitoring.v3.ProjectName;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+// Sample to create an alert policy
+public class CreateAlertPolicy {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String alertPolicyName = "your-policy-name";
+    createAlertPolicy(projectId, alertPolicyName);
+  }
+
+  public static void createAlertPolicy(String projectId, String alertPolicyName)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      ProjectName name = ProjectName.of(projectId);
+
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
```

Comment:
Would this be better using string format? or some other way to not be quoting so much?

Reply:
Done

Reply:
Not quite what I was thinking.  Users are going to want to customize this and you have lots of `\"` embedded within a string.  The first thing I found when I Google'd was this idea: https://stackoverflow.com/questions/15305956/how-to-add-quotes-around-printed-string
```java
String metricFilter = "metric.type=" 
          + '"' 
          + "compute.googleapis.com/instance/cpu/utilization" 
          + '"' 
          + " AND resource.type="
          + '"'
          + "gce_instance"
          + '"';
```

What I'm asking for is that we make it more readable and easier for users to modify and adjust.

Reply:
Done